plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -76,13 +76,30 @@ func isVolumeConflict(volume api.Volume, pod *api.Pod) bool {
 			}
 		}
 	}
+	if volume.RBD != nil {
+		mon := volume.RBD.CephMonitors
+		pool := volume.RBD.RBDPool
+		image := volume.RBD.RBDImage
+
+		manifest := &(pod.Spec)
```

Comment:
Although I suppose it safest to keep it the same as the others and change them all at one time in another PR.